Sources/SwURL/RemoteImage/RemoteImage.swift

```diff
@@ -13,7 +13,7 @@ import Combine
 @available(iOS 13.0, *)
 class RemoteImage: BindableObject {
 
-    var didChange = PassthroughSubject<Image?, Never>()
+    var willChange = PassthroughSubject<Image?, Never>()
```

Comment:
@JS1010111 is this simply just a different syntax / naming?  have you checked for behaviour changes?